File: app/monitoring/Counters.scala
package monitoring

import com.codahale.metrics.SharedMetricRegistries
import play.api.Play._

object Counters {
  def recordClaimRenderCount() {
    SharedMetricRegistries.getOrCreate(current.configuration.getString("metrics.name").getOrElse("default")).counter(s"rs-render-count").inc()
  }
}

File: src/main/java/monitoring/Counters.java
package monitoring;

import com.codahale.metrics.SharedMetricRegistries;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

/**
 * Created by peterwhitehead on 04/05/2016.
 */
@Component
public class Counters {
    @Value("${metrics.name}")
    private String metricsName;

    public void recordClaimRenderCount() {
        SharedMetricRegistries.getOrCreate(metricsName).counter("rs-render-count").inc();
    }
}
